sechub-doc/src/docs/asciidoc/documents/shared/configuration/sechub_config.adoc

```diff
@@ -292,6 +292,24 @@ include::sechub_config_example8_web_scan_openapi_with_data_reference.json[]
 <2> web scan uses "openApi" as API type
 <3> web scan uses the referenced <<sechub-config-data-section,data>> configuration "open-api-file-reference"
     to obtain the open api configuration file
+    
+[[sechub-config-example-webscan-header]]
+====== Example Header scan
+[source,json,title="header scan"]
+----
+include::sechub_config_example15_web_scan_header.json[]
```

Comment:
The numbers are not linked inside the example json 
![image](https://github.com/mercedes-benz/sechub/assets/11644753/de641bfc-8a8b-4a1d-9038-c82ef337b35b)

I will mention this inside the json file as well